libstdc++ library missing for C++ apps in docker

Some C++ library missing in docker.

https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/20996/1/console

https://github.com/zephyrproject-rtos/zephyr/pull/9533

```
[100%] Linking CXX executable zephyr.elf
/usr/bin/x86_64-linux-gnu-ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/6/libstdc++.so when searching for -lstdc++
/usr/bin/x86_64-linux-gnu-ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/6/libstdc++.a when searching for -lstdc++
/usr/bin/x86_64-linux-gnu-ld: cannot find -lstdc++
collect2: error: ld returned 1 exit status
```

--- a/shippable/install.sh
+++ b/shippable/install.sh
@@ -59,7 +59,7 @@ sudo pip install awsebcli=="$AWSEBCLI_VERSION"
 echo "======================= Installing gcc 6 ======================"
 add-apt-repository ppa:ubuntu-toolchain-r/test
 apt-get update
-apt-get install gcc-6 g++-6
+apt-get install gcc-6 g++-6 gcc-6-multilib g++-6-multilib
 update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 50 --slave /usr/bin/g++ g++ /usr/bin/g++-6
 update-alternatives --install /usr/bin/gcov gcov /usr/bin/gcov-6 50
 gcc --version
@@ -113,7 +113,6 @@ apt-get install -y \
 	cpio \
 	gperf \
 	python3-pip \
-	gcc-6-multilib \
 	gcovr \
 	valgrind \
 	ninja-build \

